### ISSUE TITLE
Feature Request: VS2019

### ISSUE THREAD

**Issue by ohlawdie** [ADN-DevTech/Maya-Net-Wizards#4]
Hey @autodesk-adn, is there any possibility we'll get this updated in the new year for VS2019? At any rate, thanks!!

![](https://media3.giphy.com/media/9QrNWBKvBpCw0/giphy.gif)
:]

**Comment by iamsleepy** [comment]
Hi, 

I've tested it with VS2019, it still works. Since the change in VS2017, it won't be shown in the C# section. If you choose all languages and type "Maya" in the search field, you can find it.

Yours,
Li

**Comment by kevinvandecar** [comment]
Ok, cool, so we just need to document the change to how to find it in the VS IDE. Li can you update readme for it? You could also blog about it to bring awareness…

I see there is also a readme.docx. I think we should consolidate anything there into the markdown (md) file and remove the docx file as well. Currently they are different, but have similar content.

Thanks,
Kevin

**Comment by ohlawdie** [comment]
Much obliged thanks for the response(s)! There is a Maya course on Pluralsight that references this template; I didn't manage to get the course material for it while I still had my monthly subscription...this will help. Cheers

**Comment by netlander** [comment]
@evanricard yes I've noticed the course on Pluralsight and was wondering if I should signup to take a look but I'm still worried that it's probably very basic. Can you comment on the Pluralsight course? Does it go in depth into the available C# APIs and the underlying C++ stuff or would you say it's for beginners?

**Comment by ohlawdie** [comment]
@netlander the author Josh Gaines does a terrific job of explaining the API thoroughly it is a pretty involved course but I think it is manageable for beginners who are getting into intermediate territory.

**Comment by ohlawdie** [comment]
@iamsleepy unfortunately I have to report that this is still not working for me in 2019. Template / items are not showing up in the respective places either when making a new project or adding a new item. I also went back to Pluralsight with a trial account and referred to the aforementioned video to get a look at what the author did to start using these templates. No surprise, he didn't do anything different than what is stated in the Readme but he did make a "fixed" version of the templates for 2019 and that still is not working for me either.

![image](https://user-images.githubusercontent.com/30779615/71785047-0f4f2b00-2fc9-11ea-9f2b-41584177d351.png)
![image](https://user-images.githubusercontent.com/30779615/71785052-170ecf80-2fc9-11ea-9a0b-17651055674e.png)

To clarify, I took both zipped and unzipped versions of the .vstemplate folders and put them into ~\Documents\VS2019\Templates\{ProjectTemplates}/{ItemTemplates}. 

![image](https://user-images.githubusercontent.com/30779615/71785166-78836e00-2fca-11ea-836d-4c84f618ecae.png)
Still not seeing anything for 'Maya' ^ 

Regardless, I could probably just manually import respective files into projects and make templates out of those.

**Comment by iamsleepy** [comment]
@evanricard that's weird. It reminds me that sometimes windows will block files downloading from internet. I built and tested the templates with my computer, so it might not be blocked.

**Comment by ohlawdie** [comment]
@iamsleepy indeed strange I'll report back if I find anything new and exciting...

**Comment by ohlawdie** [comment]
![image](https://user-images.githubusercontent.com/30779615/71832195-711a9e00-3078-11ea-9f6f-960f19d6c365.png)
Templates are working! Didn't do anything except have the computer go to sleep...and there it was.

**Comment by Andrei-2k** [comment]
Does this work for Maya 2018 or only through 2017 like it states in the description?

**Comment by iamsleepy** [comment]
Hi, @Andrei-2k . Since the style of .Net API hasn't been changed a lot and .net frameworks have great compatibility, I think it should be working fine with Maya 2018+. Please let me know if it wouldn't work.

**Comment by AGuyCoding** [comment]
@evanricard may i ask you if you still have those "fix" files around?
since it would be also in a trial account, i guess the share is not illegal? (if it is illegal i am very sorry, please ignore it then)

**Comment by ohlawdie** [comment]
I actually don't believe I have the files on hand still ; it doesn't seem that it would be illegal but I'm not absolutely _positive_..let me at least try to find the course for you and you could make a trial account.

https://www.pluralsight.com/courses/maya-2019-introduction-csharp-plugin-development

**Comment by cyrillef** [comment]
There is no thing illegal updating the code and sharing. This project is open source. And you can actually submit PRs to share with the community. I'll ask @JohnOnSoftware and Li to take a look to this one.

**Comment by kevinvandecar** [comment]
Yes, as @cyrillef says the files here are opensource and can easily be shared. In fact you can even suggest contributions / fixes if you find problems and we can review / merge the changes.
It's not clear, but I think there was also a conversation about pluralsight files from the Maya .NET plugin training. Of course pluralsight owns those files, so be sure to check in case there could be an issue. I doubt it, but we are not responsible for files outside the github source code provided here. Hope it helps.
